src/libs/CompanionDataManager.lua

```diff
@@ -546,6 +548,11 @@ local CC_COMPANION_DATA = {
                 time = CC_UNKNOWN_TIME,
                 rapport = 500,
             },
+            {
+                text = CC_GOOD_SHARP_TEXT_9,
```

Comment:
Added World Boss daily quest in Necrom from Ordinator Nelyn to Sharp-as-Night list of rapport gains